### DISCUSSION TITLE
Inpainting and Flux

### DISCUSSION THREAD

**Discussion by thiagojramos** [lllyasviel/stable-diffusion-webui-forge#2279] [General]
How are you handling inpainting with Flux models?

**Comment by Vagrin** [comment]
> How are you handling inpainting with Flux models?

Inpainting works for me just fine with Flux models. Has for quite some time. Is there something specific you're after?

**Reply by thiagojramos** [reply]
> > How are you handling inpainting with Flux models?
> 
> Inpainting works for me just fine with Flux models. Has for quite some time. Is there something specific you're after?

<details>
<summary>Some difficulties I've encountered</summary>

The difficulties I've encountered are in inpainting people who are sideways in the image and adding some objects they are holding. And even inpainting to add hair to the legs (and arms) of a man wearing shorts, for example (for this I had to resort to Photoshop's generative fill).

Another one I also have a lot of difficulty with is synchronizing glances on new faces, for example, two people in the image and one looking at the other (but I find this difficult even in SD models)

The worst is that I don't even have a concrete example that can be reproduced because I spent the week changing the parameters and models with each generation.

Another was with the soft inpaint issue that I couldn't handle properly.

</details>
---

But I opened this topic to see people talk a little bit about how the use of inpainting with flux is going, and without being able to use ControlNet.

**Reply by SunGreen777** [reply]
Well, try to describe it, the man (girl) on the left looks with loving eyes at the man (girl) on the right, and the man (girl) on the right looks with loving eyes at the man (girl) on the left. The characters look at each other with a loving gaze.

**Comment by MigCurto** [comment]
No problems for me with inpaintinf in Flux, altought I usually use inpaint upload because I can do easy rough masks on Photoshop and copy paste to drop them in Forge. But its allways hard when you need two diferent things: man/woman , dog/cat ,etc... sometimes gives you 2 of each...

I havent tested it, but there was this working for Forge: https://github.com/Haoming02/sd-forge-couple
